ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import pytest
+import pathlib
+import textwrap
+
+logger = logging.getLogger("testrunner")
+
+
+def test_dockercaps(deployment, kubectl):
+
+    CONTENT = """---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: leap
+spec:
+    containers:
+    - name: app
+      image: opensuse/leap:latest
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle12sp4
+spec:
+    containers:
+    - name: app
+      image: registry.suse.com/suse/sles12sp4:latest
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle15
+spec:
+    containers:
+    - name: app
+      image: registry.suse.com/suse/sle15:latest
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle15sp1
+spec:
+    containers:
+    - name: app
+      image: registry.suse.de/suse/containers/sle-server/15/containers/suse/sle15:15.1
+      command: ['/bin/sh', '-c', 'sleep 3600']"""
+    cwd = pathlib.Path.cwd()
```

Comment:
Creating a new file on the filesystem during testing should always be done in a manner that will ensure it is cleaned up, after the test is done.

Usually this is achieved by using the `setUp`, `tearDown` methods in `xUnit` type of frameworks - in pytest this is not considered the goto method though (https://docs.pytest.org/en/latest/xunit_setup.html) - they advise putting code like this into a `fixture` (https://docs.pytest.org/en/latest/fixture.html#fixture)

Reply:
Thanks for noticing, indeed. Now it works :)